scripts/typescript_checks.py

```diff
@@ -415,6 +415,8 @@
     'core/templates/pages/topic-editor-page/modal-templates/create-new-story-modal.controller.spec.ts',
     'core/templates/pages/topic-editor-page/modal-templates/create-new-subtopic-modal.component.spec.ts',
     'core/templates/pages/topic-editor-page/modal-templates/create-new-subtopic-modal.component.ts',
+    'core/templates/pages/topic-editor-page/modal-templates/rearrange-skills-in-subtopics-modal.component.spec.ts',
```

Comment:
[Failures](https://github.com/oppia/oppia/actions/runs/3210648838/jobs/5248255331) excluding the spec file here, it would be better them to cover under separate PR or under GSoC project
cc\ @JeeveshGarg